indexing-service/src/main/java/org/apache/druid/indexing/common/actions/RetrieveUsedSegmentsAction.java

```diff
@@ -73,35 +68,22 @@ public class RetrieveUsedSegmentsAction implements TaskAction<Collection<DataSeg
   @JsonCreator
   public RetrieveUsedSegmentsAction(
       @JsonProperty("dataSource") String dataSource,
-      @Deprecated @JsonProperty("interval") Interval interval,
```

Comment:
Is there ever a case where the payload is being sent this key during upgrade/downgrade ?

Reply:
No, this argument has been deprecated since 2015 and has not been used in the code since.